tap_github/client.py

```diff
@@ -195,32 +208,37 @@ def authed_get(self, source, url, headers={}, stream="", should_skip_404 = True)
         """
         Call rest API and return the response in case of status code 200.
         """
-        with metrics.http_request_timer(source) as timer:
+        with metrics.http_request_timer(url) as timer:
```

Comment:
This is already expecting the endpoint, it makes it easier to identify the request in logs.